crypto/build.rs

```diff
@@ -0,0 +1,14 @@
+//! Build script
+//!
+//! Warn if `ffi_import` and `ffi_export` features are active at the same time
+
+fn main() {
+    let ffi_import = std::env::var_os("CARGO_FEATURE_FFI_IMPORT").is_some();
```

Comment:
I'm with @Erigara on this one. It should be done with a `compile_error`.

Reply:
`compile_error` would cause the compilation to fail. The check in the build script only produces warning. I can move the warning from build script to the code if you would prefer that.

Is there some other way that I am not aware of to implement feature precedence?

Reply:
@appetrosyan, currently, as @mversic said in chat, our ci uses `--all-features` which will always fail in case of `compile_error`.
Cool that `cargo-all-features` could skip incompatible sets of features.

Reply:
Features are not incompatible. `build.rs` only warns that precedence is taking place. How would I print the warning otherwise?

Reply:
@appetrosyan are we using `cargo-all-features` yet? because in the case of precedence we will never test one of the features with `--all-features`

Reply:
Cargo all features is orthogonal to `cargo test --all-features`, with good reason. We only need to check feature combinations once every release. Making all tests take longer is a bad idea. 

Resolving.